indexing-hadoop/src/main/java/io/druid/indexer/InputRowSerde.java

```diff
@@ -65,7 +67,8 @@
   {
     ValueType getType();
 
-    void serialize(ByteArrayDataOutput out, Object value, boolean reportParseExceptions);
+    @Nullable
+    String serialize(ByteArrayDataOutput out, Object value, boolean reportParseExceptions);
```

Comment:
I'm not sure about returning an exception message. Probably it's possible that this method always throws a `ParseException` on parsing errors and `reportParseExceptions` can be handled by the caller.

Reply:
Changed this throw parse exception instead of returning the message